### ISSUE TITLE
Transport protein in red blood cells

### ISSUE THREAD

**Issue by teutonicod** [Open-Systems-Pharmacology/Forum#266]
Dear all,

I would like to create a transporter in PK-Sim which is expressed in different tissues, and red blood cells is one of them.
Is there a reason why enzymatic processes and binding partners can be expressed in red blood cells (or the vascular endothelum) but transporters cannot? Since red blood cells are not available in the list of tissues where the transporters can be expressed. 

Best regards,

Donato

**Comment by JGWojtyniak** [comment]
Dear Donato,

As far as I know, there is no specific reason why transporters cannot be expressed in red blood cells in PK-Sim. However, you can implement them in MoBi. You can find a detailed description how in the following post: https://github.com/Open-Systems-Pharmacology/Forum/issues/248

Good luck and best regards,
Jan

**Comment by prvmalik** [comment]
Confirming @teutonicod that transporters cannot be expressed in RBC in PKSim.
Same options available for influx, efflux, pgp-like.
![for github](https://user-images.githubusercontent.com/29066045/50846616-9a0d3380-133d-11e9-88ee-ae10a50753f4.JPG)

**Comment by msevestre** [comment]
@all: The limitation is by no mean technical so I am assuming this was a request at the time to NOT offer this type of transports.

Can you describe how such transports would look if implemented in PKSim in the case of Efflux, Influx and Pgp e.g. Source and Target of transport process ?

**Comment by PavelBal** [comment]
>Can you describe how such transports would look if implemented in PKSim in the case of Efflux, Influx and Pgp e.g. Source and Target of transport process ?

**Influx** - Plasma to RBC
**Efflux** - RBC to plasma

**Pgp** - no idea, would not make sence in my opinion.

**Comment by teutonicod** [comment]
I agree with @PavelBal for the Red blood cells. I think it would be worth it also to discuss the same point for the other components of the vascular system which are not available for the transporters:
I prepared a summary table below so that it could be used as base to the discussion:

- **RBC** 
_Influx_ - Plasma to RBC
_Efflux_ - RBC to plasma
_Pgp_ - Does it make sense?

- **Plasma**--> I don't think that it make sense for the transporters

- **Vascular endothelium** -->  I think that this could be useful for macromolecules presenting an active transport across the vascular endothelium
_Influx_ - Plasma to interstitial space
_Efflux_ - Interstitial space to plasma
_PgP_ - Does it make sense?

**Comment by msevestre** [comment]
Let's move this issue to the PKSim repo as a feature request. Thanks everyone
